src/tests/ngraph_helpers/snippets_ngraph_functions/src/function_helper.cpp

```diff
@@ -0,0 +1,73 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "function_helper.hpp"
+#include "common_test_utils/data_utils.hpp"
+#include <snippets/snippets_isa.hpp>
+#include <snippets/op/subgraph.hpp>
+#include "ngraph_functions/builders.hpp"
+
+namespace ov {
+namespace test {
+namespace snippets {
+
+// TODO: workaround while element-wise operations after `Parameter` are not added in Subgraph
```

Comment:
Could you please remind me why can't we enable subgraphs after inputs at this point? Do we need to support some extra precisions?